cognee/modules/data/processing/detect_chunk_language.py'.py

```diff
@@ -0,0 +1,46 @@
+import boto3
```

Comment:
There is `'` in filename, probably mistake.

Reply:
Do we want to detect it on the document level, or on the chunk level?

Reply:
How would tasks work on document level vs chunk from design side?